### ISSUE TITLE
Dostarczanie maili do Poradni

### ISSUE THREAD

**Issue by PiotrIw** [watchdogpolska/poradnia#1253]
@AgnieszkaZdanowicz @KarolaJ @mrbojko 
Zauwarzyłem że reguły na wejściu do poradni odrzucają maile jeśli:
a) prawidłowy adres typu "Szymon Osowski <sprawa-8963@porady.siecobywatelska.pl>"  jest w "cc" zamiast w "to"
b) adres ma postać "Szymon Osowski <main@porady.siecobywatelska.pl>" - w sumie nie wiem czy prawidłaową i skąd sie mogą takie brać

Czy to jest zachowanie znane i oczekiwane czy nie?

**Comment by KarolaJ** [comment]
@AgnieszkaZdanowicz co do dawania poradni w cc - to zdecydowanie tak. :)

**Comment by PiotrIw** [comment]
@AgnieszkaZdanowicz  
....co do dawania poradni w cc - to zdecydowanie tak. :) - zdecydowanie tak czyli rozumiem że oczekiwane?

Jeśli tak to rozumiem ideę. Jednak to w tej chwili działa tak że jak ludzie dają poradnię w cc, to taki mail jest wysłany, potem jest przyjęty do skrzynki w zenboxie ale odrzucony przez poradnię, czyli nadawca wie że mail nie został odrzucony ale w poradni go nie widać. 

Może zamiast odrzucać maile pomiędzy poradnią a zenboxem, lepiej byłoby na zenboxie założyć formułę z autoreplay'em cc typu "ten mail nie został przyjęty...." a czego nie wychwyci formuła to niech jednak trafia na poradnię. Przynajmniej będziemy mieli ten sam stan wysyłki/odbioru co nadawca? 

Hmm, to również będzie tylko częściowo skuteczne bo głównych odbiorców może być wielu, więc jeśli ludzie, wysyłając pisma do urzędów, dadzą poradnię jako drugiego adresata to zenbox nie odrzuci.....

**Comment by AgnieszkaZdanowicz** [comment]
@PiotrIw Tak, chodzi o utrzymanie tego mechanizmu. Tak, jak proponujesz byłoby najlepiej, żeby te wiadomości, gdzie jesteśmy na cc, w ogóle nie wpływały na zenboxa i myślałam, że tak właśnie jest, szczerze mówiąc. Nie wiem, po co trzymamy je na zenbox, skoro i tak nikt z nimi nic nie zrobi...

Co do ostatniego akapitu (o kilku głównych odbiorcach) to rozumiem, że teraz też to tak działa, że takie wiadomości trafiają na poradnię? Chyba nie zalewają nas takie wiadomości, więc możemy utrzymać ten stan. @KarolaJ zgadza się?

**Comment by KarolaJ** [comment]
@AgnieszkaZdanowicz Tak, zgadza się. 

@AgnieszkaZdanowicz  @PiotrIw a wczoraj dostaliśmy taka wiadomość, gdzie byliśmy w kopii (cc) i dotarło, zobaczcie tutaj: https://porady.siecobywatelska.pl/sprawy/sprawa-15275/#letter-77852

**Comment by AgnieszkaZdanowicz** [comment]
@KarolaJ mi się wydaje, że Pan próbował nam wysłać w kopii, ale może zobaczył na poradni, że się nie wyświetliło i przeforwardował nam tę wiadomość. Bo gdyby to była po prostu kopia, to byśmy nie widzieli tego nagłówka

**Comment by KarolaJ** [comment]
@AgnieszkaZdanowicz rzeczywiście, masz racje :)

**Comment by AgnieszkaZdanowicz** [comment]
@PiotrIw czy tutaj zawiesiliśmy temat - znaczy, czy wiadomości do CC nadal spływają na zenboxa czy są jednak odrzucane? Nie wiem, na ile to jest duża robota. Bo może to też wpisać do projektu i uwzględnić w kosztach?

**Comment by PiotrIw** [comment]
expected logic
